inline/tests/lexer/snapshot.rs

```diff
@@ -43,9 +43,17 @@ impl AsSnapshot for Snapshot<Token> {
         // only newline token spans 2 lines, should not be the case for others!
 
         let inner = match self.as_str() {
-            "\n" => "\u{23CE}",
+            "\n" => Self::NEWLINE_SYBMOL,
             other => other,
         };
+
+        if span.len_utf8().unwrap_or(1).saturating_sub(inner.len()) == 1 {
+            // escaped token's occupy two characters in text (the backslash and symbol). In such
+            // cases, span is longer than the actual content by a single character.
+            // Push content to the right "\*" will be rendered as " *"
```

Comment:
```suggestion
            // e.g. content like "\*" will be rendered as "␢*" in snapshots to indicate the backslash escape.
```

Just to clarify that we only render it like that for testing.